http/gateway/error.go

```diff
@@ -49,21 +49,27 @@ func (h *ErrorHandler) ErrorxProtoErrorHandler(ctx context.Context, mux *runtime
 func (h *ErrorHandler) httpResponse(ctx context.Context, err error, r *http.Request, logMd log.Metadata) (interface{}, int) {
 	s, ok := status.FromError(err)
 
-	internalErr := &errorx.Error{
+	defaultErr := &errorx.Error{
 		Code:    errorx.CodeInternal,
 		Message: err.Error(),
```

Comment:
don't forget to use message from `s`, so that the message from internal gateway error is propagated to `defaultErr`

using `err.Error()` will produce error that has prefix like: `rpc error code: 3, some error happened`

using `s.Message()` will produce `some error happened`